spec/support/reference_data.rb

```diff
@@ -25,6 +25,14 @@
   config.before(:all) do
     SubjectCreatorService.new.execute
     SubjectFinancialIncentiveCreatorService.new.execute
```

Comment:
I guess the SubjectFinancialIncentiveCreatorService and SubjectCreatorService be migrated over to the TestSetup pattern? And this is where sub-classing the TestSetup class might be useful ...

Reply:
Well, the thing is that `SubjectCreatorService` and `SubjectFinancialIncentiveCreatorService` are used by the db seeding. I wouldn't want db seeding to use factories, so moving them here would imply duplicating whatever logic is in those services between seeding and factories.